pandas/util/testing.py

```diff
@@ -893,11 +912,13 @@ def assert_attr_equal(attr, left, right, obj='Attributes'):
         Attribute name being compared.
     left : object
     right : object
-    obj : str, default 'Attributes'
-        Specify object name being compared, internally used to show appropriate
+    obj : str or 2-length sequence (tuple, list, ...), default 'Attributes'
```

Comment:
this is not correct, in any event you can just accept a list here (only), this is an internal routine